src/consensus/pbft/libbyz/Replica.cpp

```diff
@@ -2004,71 +2207,23 @@ bool Replica::execute_tentative(Pre_prepare* pp, ByzInfo& info)
 
     while (iter.get(request))
     {
-      int client_id = request.client_id();
-
-#ifdef ENFORCE_EXACTLY_ONCE
```

Comment:
This seems to have gone in the refactoring. I guess we were not using it anyway/are not planning to use it?

Reply:
Yeah it is not being used. I haven't removed it consistently from all of the code base but where I have refactored a bit I have removed it